docs/competition/matches/match-scoring.md

```diff
@@ -18,36 +18,51 @@ Note that this role is _not_ about making decisions related to the interpretatio
+You are not responsible for:
+- Making sure a match starts with the correct number of teams -- that is down to the shepherds.
 
-If the team members indicates that they wish for you to turn off their robot (i.e. "throwing in the towel") you may do so but by doing so they forfeit the match which will be recorded in the score sheets as a disqualification. There are two cases where you are allowed to switch off a robot yourself, one is when the match ends and the other is when a robot is causing **unintentional and significant** damage to another robot (i.e. mashes into another robot, pins it, stopping it from being able to function -- includes one dragging the other around if they become intertwined), and in that case it is up to you to decide which robot should be switched off but their match is not forfeited. If the damage is intentional, then the team will get forfeited (potentially for previous matched too), but this decision should be left to the judge. A checklist for robot damage is available below.
+### Scoring
 
-You must read [the rules](https://www.studentrobotics.org/docs/rules/) before starting this role to make sure you understand the game, the scoring rules and how the arena is laid out.
+Both marshalls will be given a score sheet on a clipboard, which you must both individually fill in fully by the end of the match. You must do your best to record the state as it is at the exact end of the match. Anything that happens after the match has ended must not be recorded, and the arena must not be changed until both scorers have finished. Teams are not allowed to enter the arena to collect their robots until after scoring has finished.
 
+Once you have finished scoring, you must drop your completed score sheet at the score entry desk which will be near the arenas. You must return promptly to the arena in case you need to be found for further scoring questions, and begin resetting the arena for the next match.
+
+#### Teams touching robots
+
+Teams must not be allowed in the arena during a match. It is the responsibility of the team members to place their robot in the start zone before the match starts. If a robot moves off before the match starts (aside from minor servo-resetting arm movements) teams are allowed to switch off their robot, reset it and start again. However, if the mis-started robot touches any arena prop then they should be immediately disqualified (there is a checkbox on the score sheet). Once the robot start button has been pressed and the match has started, teams are not allowed to interact with their robots until after scoring has taken place.
+
+#### Turning off robots
```

Comment:
We need a ruling from @prophile about whether we're going to allow this given the robot-location-bouns in this year's game.

Reply:
This doesn't matter too much due to the disqualifications when throwing in the towel.